cloudwatch/cloudwatch.go

```diff
@@ -84,6 +88,12 @@ type Event struct {
 	stream string
 }
 
+//TaskMetadata is the task metadata of a task
+type TaskMetadata struct {
+	Cluster string
+	TaskARN string
+}
```

Comment:
Can you add Task ID to this structure. You can unmarshal the response into this structure, parse the task ID, then set it here. 

This is just for code organization and readability. It feels weird to have this structure that says "TaskMetadata is the task metadata of a task" but then have task Id be separate.

Reply:
yeah make sense.